src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -280,6 +280,24 @@ public void FilterRules_ValidConfig_RulesFiltered(bool isBicep, string configNam
             }
         }
 
+        [DataTestMethod]
+        [DataRow("nestedOnlyDefinedProperties.json", DisplayName = "Variable/Parameter name not found warning should not be generated")]
+        public void ProcessTemplateResourceLanguageExpressions_PropertiesDefinedInInnerTemplateOnly_NoWarning(string relativeTemplatePath)
```

Comment:
```suggestion
        public void AnalyzeTemplate_PropertiesDefinedInInnerTemplateOnly_NoWarning(string relativeTemplatePath)
```

Reply:
This test is validating behavior in Analyzer.Core, not in the CLI.  Please move the test to the Analyzer.Core test project.  (Also, tests with only a single variation shouldn't be a DataTestMethod.  It should be a standalone test with no parameters.